telemetry-proxy/Dockerfile

```diff
@@ -0,0 +1,10 @@
+# Need ubuntu for dynamic linking
+FROM ubuntu:22.04 AS run
+
+# Copy the build output from the GitHub Workflow to Alpine
```

Comment:
Idk how much it matters if we have the right comment, so feel free to just resolve this and I'll approve if you don't want to change it :)

Reply:
Lol, it's worth doing it right I think. Updated